docs/core/tools/dotnet-new.md

```diff
@@ -53,79 +60,163 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
 
 The template to instantiate when the command is invoked. Each template might have specific options you can pass. For more information, see [Template options](#template-options).
 
+If the `TEMPLATE` value isn't an exact match on text in the **Templates** or **Short Name** column, a substring match is performed on those two columns.
+
+# [.NET Core 2.2](#tab/netcore22)
+
+The command contains a default list of templates. Use `dotnet new -l` to obtain a list of the available templates. The following table shows the templates that come pre-installed with the .NET Core SDK 2.2.100. The default language for the template is shown inside the brackets.
+
+| Templates                                    | Short Name        | Language     | Tags                                  |
+|----------------------------------------------|-------------------|--------------|---------------------------------------|
+| Console Application                          | `console`         | [C#], F#, VB | Common/Console                        |
```

Comment:
Nit: Backticks on the short names.

Reply:
@mairaw I assume we want to keep the **Short Name** column values in backticks to avoid localization. Is that correct?

Reply:
yes @scottaddie